agenda/data/src/main/java/com/nibalk/tasky/agenda/data/local/entity/EventPendingSyncEntity.kt

```diff
@@ -0,0 +1,22 @@
+package com.nibalk.tasky.agenda.data.local.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = EventPendingSyncEntity.TABLE_NAME)
+data class EventPendingSyncEntity(
+    @Embedded val event: EventEntity,
+    @PrimaryKey(autoGenerate = false)
+    @ColumnInfo(name = PRIMARY_KEY)
+    val eventId: String = event.id,
+    val localPhotos: List<ByteArray>,
```

Comment:
As per requirements, saving local photos is not required. Also if you'd implement that I'd be rather careful storing this much data in a single DB field. The more typical approach would be saving the file on the file system and just saving the references to it in the DB.